rust/opendp_tooling/src/codegen/python.rs

```diff
@@ -200,8 +200,16 @@ def {then_name}(
 
 {doc_params}{example}
     """
-    return PartialConstructor(lambda {dom_met}: {name}(
+    output = PartialConstructor(lambda {dom_met}: {name}(
 {args}))
+    output.__opendp_dict__ = {{
+            'func': '{then_name}',
+            'module': 'measurements',
```

Comment:
This is hardcoded when it should be based on the module, `module_name` is passed in above.

Reply:
```suggestion
            'module': '{module_name}',
```